index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home - Ghana Digital Innovation Week</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+</head>
+<body class="body" onload="displayWorks()">
+    <section class="desktop-header">
+        <div class="grid-topnav container">
+            <div>
+                <a href="http://" class="topnav-items"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="" class="topnav-items"><i class="fa-brands fa-twitter"></i></a>
+                <span><a href="" class="topnav-items">English</a></span>
+                <span><a href="" class="topnav-items">Twi</a></span>
+                <span><a href="" class="topnav-items">Dagbani</a></span>
+            </div>
+        </div>
+        <div class="flex-navbar container">
+            <div>
+                <a href="index.html"><img src="img/Nov-GDIW-LOGO.svg" alt="GDIW Logo"></a>
+            </div>
+            <div class="nav-items">
+                <ul>
+                    <li><a href="index.html" class="active">Home</a></li>
+                    <li><a href="about.html">About Conference</a></li>
+                    <li><a href="about.html/#past-events">Past Events</a></li>
+                </ul>
```

Comment:
Kindly wrap this inside a `nav` tag. It is semantically correct to wrap navigational links inside a `nav` tag. :nerd_face: :nerd_face: . Please change this also on the about page.